subjects/java/checkpoints/time-tracker/README.md

```diff
@@ -0,0 +1,72 @@
+## TimeTracker
+
+### Instructions
+
+Write a `ProjectTime` class to track the duration of a project using start and end times. You will be asked to add getter and setter methods for the `startTime` and `endTime` attributes. When you set one of them, you must update the `hoursLogged` attribute. The `hoursLogged` getter should output the hours in the following format:
```

Comment:
What is the format accepted for `startTime` and `endTime`?

Reply:
the accepted format is clearly showed in the examples